### PR TITLE
[C-1352] Fix twitter deep link

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Was bothering me... Required a bit more stuff per latest instructions
https://reactnavigation.org/docs/deep-linking/

Nice one-line tester tho
```npx uri-scheme open "audius://" --ios```

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

```npx uri-scheme open "audius://" --ios```

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Review by sliptype — APPROVED** [review]
Amazing! Thank you for knocking this out

File: packages/mobile/ios/AudiusReactNative/Info.plist [hunk 1]
```diff
@@ -1,106 +1,110 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>en</string>
+	<key>CFBundleDisplayName</key>
+	<string>${APP_DISPLAY_NAME}</string>
```
Comment (dylanjeffers) [inline packages/mobile/ios/AudiusReactNative/Info.plist:8]: Mind giving me the quick info about changes for this file?

File: packages/mobile/src/components/navigation-container/NavigationContainer.tsx [hunk 2]
```diff
@@ -34,8 +34,10 @@ const NavigationContainer = (props: NavigationContainerProps) => {
 
   const linking = {
     prefixes: [
+      'audius://',
```
Comment (dylanjeffers) [inline packages/mobile/src/components/navigation-container/NavigationContainer.tsx:37]: Nice

**Review by dylanjeffers — APPROVED** [review]
Thanks for handling

File: packages/mobile/ios/AudiusReactNative/Info.plist (hunk 1 above)
Comment (raymondjacobson) [reply packages/mobile/ios/AudiusReactNative/Info.plist:8]: xcode auto formatted most of it. i just added co.audius to the universal link section :(

**Pull request merged**